src/Libraries/Nop.Core/Domain/Security/HashFormat.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nop.Core.Domain.Security {
+    public enum HashFormat {
+        Sha1 = 0,
```

Comment:
I'm not a maintainer here, but why add support for SHA1?  It has been broken in practice, in some cases, and is not being used by governments any more.   I'd suggest removing this from the set of potential hash formats.